dataeng/jobs/analytics/SnowflakeQueueDepth.groovy

```diff
@@ -0,0 +1,75 @@
+package analytics
+
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm
+import static org.edx.jenkins.dsl.AnalyticsConstants.secure_scm_parameters
+
+
+class SnowflakeQueueDepth {
+"""
+This job runs every 10 minutes using the Snowflake task automation user.
+"""
+    public static def job = { dslFactory, allVars ->
+
+        Map SnowflakeQueueDepthConfig = [
+            NAME: "collect-queue-depth",
+            MANUAL: false,
+            SCRIPT_TO_RUN: "dataeng/resources/snowflake-collect-queue-depth.sh"
+        ]
+        List jobConfigs = [
+            SnowflakeQueueDepthConfig
+        ]
```

Comment:
I think this whole section can be deleted provided that you moved the strings into the code below.